src/histogram_equalization.jl

```diff
@@ -287,7 +304,7 @@ end
 Histogram equalization of `img` using `nbins` bins.
 """
 function histeq(img::S; nbins=64)::S where {S<:AbstractArray{<:Integer}}
```

Comment:
Does this always need to be set to 64 bins?

Reply:
Not really but that's how MATLAB's `histeq` behaves. See [here](https://tinyurl.com/mskyx9mm).